examples/b2c/checkout/integrations/classes/B2BTaxSample.cls

```diff
@@ -78,9 +100,24 @@ global class B2BTaxSample implements sfdc_checkout.CartTaxCalculations {
                     TaxRate = rateAndAmountFromExternalService.getRate(),
                     TaxType = 'Actual'
                 );
+                
                 cartTaxestoInsert.add(tax);
+                
+                for (CIPATax cipaTax : rateAndAmountFromExternalService.getItemizedPromotionTaxAmounts()) {
+                    CartTax promoTax = new CartTax( 
+                        Amount = cipaTax.getTaxAmount(),
+                        CartItemId = cartItemIdsBySKU.get(sku),
+                        Name = rateAndAmountFromExternalService.getTaxName(),
+                        TaxCalculationDate = Date.today(),
+                        TaxRate = rateAndAmountFromExternalService.getRate(),
+                        TaxType = 'Actual',
+                        CartItemPriceAdjustmentId = cipaTax.getId()
```

Comment:
You also need to set AdjustmentTargetType

Reply:
Adjustment Target Type is derived.